warehouse/query-core/src/main/java/datawave/query/tables/edge/EdgeQueryLogic.java

```diff
@@ -257,7 +257,7 @@ protected VisitationContext normalizeJexlQuery(String query, boolean getFullNorm
         
         // These strings are going to be parsed again in the iterator but if there is a problem with
         // normalizing the query we want to fail here instead of over on the server side
-        if (!visitationContext.getNormalizedQuery().equals("")) {
```

Comment:
[ERROR] Call to StringBuilder.equals(String) in datawave.query.tables.edge.EdgeQueryLogic.normalizeJexlQuery(String, boolean) [datawave.query.tables.edge.EdgeQueryLogic] At EdgeQueryLogic.java:[line 260] EC_UNRELATED_TYPES
[ERROR] Call to StringBuilder.equals(String) in datawave.query.tables.edge.EdgeQueryLogic.normalizeJexlQuery(String, boolean) [datawave.query.tables.edge.EdgeQueryLogic] At EdgeQueryLogic.java:[line 273] EC_UNRELATED_TYPES